demo/client.ts

```diff
@@ -431,3 +452,41 @@ function serializeButtonHandler(): void {
     term.write(output);
   }
 }
+
+function initImageAddonExposed(): void {
+  const DEFAULT_OPTIONS: any = (addons.image.instance as any)._defaultOpts;
+  const limitStorageElement = (<HTMLInputElement>document.getElementById('image-storagelimit'));
```

Comment:
querySelector supports generics which makes it a little nicer:

```suggestion
  const limitStorageElement = document.querySelector<HTMLInputElement>('#image-storagelimit');
```